packages/boilerplate/composables/src/composables/getters/userBillingGetters.ts

```diff
@@ -0,0 +1,30 @@
+import { UserBillingGetters } from '@vue-storefront/core';
+
+const userGetters: UserBillingGetters<any, any> = {
+  getAddresses: (billing, criteria?: Record<string, any>) => {
+    const { addresses } = billing;
+    if (!criteria || !Object.keys(criteria).length) {
+      return addresses;
+    }
+    const entries = Object.entries(criteria);
+    return billing.addresses.filter(
+      address => entries.every(([key, value]) => address[key] === value)
+    );
+  },
+  getDefault: billing => billing.addresses.find(address => address.isDefault),
+  getTotal: billing => billing.addresses.length,
```

Comment:
Please also add `getId`, `getApartment` and `getIsDefault`.

I had to add them for `useUserShipping`.

Reply:
`getApartment` - > `getApartmentNumber`